src/app/dashboard/layout.js

```
@@ -0,0 +1,92 @@
+          <header className="navbar_dashbord shadow p-4 flex items-center justify-between">
+            <button
+              onClick={() => setIsSidebarOpen(!isSidebarOpen)}
+              className="p-2 rounded" style={{ backgroundColor: "#F77F00" }}
+            >
+              <span className="text-white fw-bold" > <TfiMenu /></span >
+            </button>
+            <h1 className="text-xl text-center font-bold">👋🏻 Bonjour, Mr <b style={{ color: "#F77F00" }}>XX</b></h1>
+
+            <div className="flex items-center gap-4" id="navbarSupportedContent">
+              <div>
+                <a href="/"> <span>⬅</span> Revenir au siteweb </a>
```

ESLint

[error] Do not use an `<a>` element to navigate to `/`. Use `<Link />` from `next/link` instead. See: https://nextjs.org/docs/messages/no-html-link-for-pages

(@next/next/no-html-link-for-pages)